source/dub/packagemanager.d

```diff
@@ -37,9 +37,10 @@ class PackageManager {
 		bool m_disableDefaultSearchPaths = false;
 	}
 
-	this(NativePath user_path, NativePath system_path, bool refresh_packages = true)
```

Comment:
Please add a new constructor, this argument is breaking the API.

Reply:
done